src/app/proveedor/proveedor.component.html

```diff
@@ -0,0 +1,126 @@
+<div class="container text-center">
+    <div class="mt-5">
+      <div class="card w-75 mx-auto">
+        <div class="card-header">
+          <h2>{{ title }}</h2>
+        </div>
+        <div class="card-body">
+          <div class="container text-center">
+            <form [formGroup]="proveedorForm" (ngSubmit)="onSubmit()">
+              <div class="row mt-3">
+                <div class="col">
+                  <label for="cedula_juridica_proveedor">Cedula Juridica</label>
+                  <input
+                    type="text"
+                    formControlName="cedula_juridica_proveedor"
+                    class="form-control mt-2"
+                    [ngClass]="{
+                      'is-invalid': trabajadorFormControls['cedula_juridica_proveedor'].errors
```

Comment:
```suggestion
                      'is-invalid': proveedorFormControls['cedula_juridica_proveedor'].errors
```

**HACER ESTE CAMBIO EN TODO EL HTML.**